crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -346,11 +346,14 @@ pub enum AdminResponse {
     /// [`AdminRequest::ListCellIds`]: enum.AdminRequest.html#variant.ListCellIds
     CellIdsListed(Vec<CellId>),
 
-    /// The succesful response to an [`AdminRequest::ListActiveApps`].
+    /// The succesful response to an [`AdminRequest::ListEnabledApps`].
     ///
     /// Contains a list of all the active `App` ids in the conductor
     ///
-    /// [`AdminRequest::ListActiveApps`]: enum.AdminRequest.html#variant.ListActiveApps
+    /// [`AdminRequest::ListEnabledApps`]: enum.AdminRequest.html#variant.ListEnabledApps
+    EnabledAppsListed(Vec<InstalledAppId>),
+
+    #[deprecated = "alias for EnabledAppsListed"]
```

Comment:
Does deprecating things work again? At one point we couldn't do this due to the `#[deny(warnings)]`...

Reply:
If you don't use a deprecated thing anywhere, then there is no warning :) Although, we do have a bunch of `#![allow(deprecated)]` sprinkled around which we should eventually get around to removing...